K/KMA/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "KMA"
+version = v"1.3.21"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://bitbucket.org/genomicepidemiology/kma.git",
+        "2d3acb03ac8f309094eee8d53eb043f033f8d8c0",
+    ),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/kma
+# Apache 2.0 license not included in the bitbucket repo
+install_license ../LICENSE
```

Comment:
We provide some common licenses in [`/usr/share/licenses`](https://github.com/JuliaPackaging/Yggdrasil/tree/5fb343ac5c89a09866678eedcedc1daccb33bebb/0_RootFS/Rootfs/bundled/licenses) for cases like this:
```suggestion
install_license /usr/share/licenses/APL
```
and you don't need to add the license file